spec/models/user_spec.rb

```diff
@@ -416,20 +416,25 @@
   end
 
   describe "self.search" do
-    it "find users by email" do
-      user1 = create(:user, email: "larry@consul.dev")
-      create(:user, email: "bird@consul.dev")
-      search = User.search("larry@consul.dev")
+    describe "find users" do
+      let(:larry) { create(:user, email: "larry@consul.dev", username: "Larry Bird") }
+      let(:robert) { create(:user, email: "robert@consul.dev", username: "Robert Parish") }
 
-      expect(search).to eq [user1]
-    end
+      it "by email" do
+        expect(User.search("larry@consul.dev")).to eq [larry]
+      end
+
+      it "by email with whitespaces" do
+        expect(User.search(" larry@consul.dev ")).to eq [larry]
+      end
 
-    it "find users by name" do
-      user1 = create(:user, username: "Larry Bird")
-      create(:user, username: "Robert Parish")
-      search = User.search("larry")
+      it "by name" do
+        expect(User.search("larry")).to eq [larry]
+      end
 
-      expect(search).to eq [user1]
+      it "by name with whitespaces" do
+        expect(User.search(" larry")).to eq [larry]
```

Comment:
How about testing trailing whitespace as well?

```suggestion
        expect(User.search(" larry ")).to eq [larry]
```